get-started/configuration.md

```diff
@@ -104,4 +108,4 @@ Once the client is configured, you can start using its features. Get to know how
 - [Create an API request via the Composer]({%slug composer-get-started%})
 - [Modify traffic]({%slug modify-traffic-get-started%})
 - [Mock server response]({%slug autoresponder-get-started%})
-- [Share captured traffic with collaborators]({%slug collaboration-get-started%}).
\ No newline at end of file
+- [Share captured traffic with collaborators]({%slug collaboration-get-started%}).
```

Comment:
(Unsure about why this got changed, I suspect there was no EOL at EOF or something.)